fuel-vm/src/interpreter/memory.rs

```diff
@@ -414,24 +414,24 @@ pub(crate) fn memcopy(
     let (bc, of) = b.overflowing_add(c);
     let overflow = overflow || of;
 
+    let dst_before_src = a <= b;
```

Comment:
Why do we need this change? Previously it was clear=)

Reply:
Because we need to know if dst is lower then src but I can try to make it clearer